docs/staking.md

```diff
@@ -0,0 +1,48 @@
+
+The pot with staking rewards is periodically distributed in its fullest to all stakers. The amount that users are entitled to is proportional to the relative size of their stake in the stake pool. However, stakers do not automatically receive the rewards on their account - instead, they need to claim them.
+
+When it comes to claiming rewards, all participants in HDX staking should be aware of the elements of **loyalty and gamification**. Once rewards are awarded, they cannot be instantly claimed for the full amount - doing so would yield just a fraction of the total rewards, with the remainder being returned the pot for redistribution to all other stakers.
+
+Users who want to claim as many rewards as possible should keep their HDX staked without claiming until sufficient time has passed (rewards are “vested” following a bonding curve). The length of the waiting period is dynamic and depends on the user (in)actions. A user who just **stakes passively** would need to **wait ~2 years to claim 95% of their rewards**. In contrast, **active stakers** who collect the maximum amount of action points (more on that below) could claim **95% of their rewards in just over 2 months.** These are rough estimates - the actual timelines may vary in accordance with user actions and overall count of referenda.
+
+## Boosting Your Rewards {#boosting-your-rewards}
+
+Stakers can increase the pace at which they can claim their rewards by **collecting action points** and **boosting their rewards**. Action points can be acquired by performing certain actions that are incentivized by the Protocol. Initially, the only way to collect action points is to **participate in the governance of HydraDX** by [voting on community referenda](https://hydradx.subsquare.io/democracy/referenda) **using the staked HDX**.
+
+<div style={{textAlign: 'center'}}>
+  <img alt="login" src={useBaseUrl('/staking/rewards_bonding_curve.jpg')} width="500px" />
+</div>
+
+There are 2 factors which determine the amount of action points that stakers will receive: The **size of the vote** (relative to the total size of their staked HDX), and the **conviction multiplier**. The higher the conviction multiplier of the vote, the greater its weight. Keep in mind that voting with a conviction multiplier places a **temporary lock on the tokens**. Stakers looking to achieve the highest rewards boost would be voting with 6x conviction multiplier, thereby locking their HDX for 192 days. Just a reminder that **this lock is not related to staking** as such - instead, it is a standard feature of governance in the Polkadot ecosystem (more info [in our docs](/democracy_referenda#referenda-votes-weighing)).
```

Comment:
perhaps it would be prudent to mention here that the 192 day lock would apply every time they vote with 6x conviction - i.e. spell out that this isn't a one-time lock of 192 days, it's 192 days from the last time they voted with conviction